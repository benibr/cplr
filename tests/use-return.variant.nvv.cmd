../cplr 'ret = 69' -nvv
