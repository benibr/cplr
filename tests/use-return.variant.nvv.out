Preparation phase
Generation phase
Generating code
Generating defsysinclude
Generating labeled statements
Generated: 1037 bytes code, 0 bytes dump
Compilation phase
Compilation succeeded
